contracts/roles/AgentManager.sol

```diff
@@ -0,0 +1,112 @@
+pragma solidity ^0.6.0;
+
+import "../token/IToken.sol";
+import "../registry/IIdentityRegistry.sol";
+import "./AgentRoles.sol";
+import "@onchain-id/solidity/contracts/IIdentity.sol";
+
+contract AgentManager is AgentRoles {
```

Comment:
Maybe we can add those permission system to the TrustedIssuerRegistry as well.

Reply:
yes, indeed, but it's not for the agent role anymore as TrustedIssuerRegistry is only `Ownable` i was planning to write an `OwnerManager` contract for all the `onlyOwner` functions

Reply:
Shouldn't the TrustedIssuerRegistry be managed by an Agent as well?

Reply:
no, it's the owner who decides who are the trusted issuers, the same way as he decides who are the agents :)